Controllers/ImageController.cs

```diff
@@ -0,0 +1,336 @@
+﻿using System.IO.Compression;
+using System.Net;
+using System.Text.RegularExpressions;
+using Newtonsoft.Json;
+
+using AppImageGenerator.Models;
+
+using SixLabors.ImageSharp.Formats;
+using SixLabors.ImageSharp.Formats.Jpeg;
+using SixLabors.ImageSharp.Formats.Png;
+using SixLabors.ImageSharp.Formats.Webp;
+using SixLabors.ImageSharp.Formats.Bmp;
+using SixLabors.ImageSharp.Formats.Tiff;
+
+using Microsoft.AspNetCore.Mvc;
+
+using System.Net.Http.Headers;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+using System.Net.Http;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace AppImageGenerator.Controllers
```

Comment:
Should use file-scoped namespace and just add `;` remove parentheses.

Reply:
Fixed